### ISSUE TITLE
Tile, base url and api prefix

### ISSUE THREAD

**Issue by aabuhasan** [athiththan11/FHIR-to-Swagger#4]
Be able to change the following.

 fhir-to-swagger Claim Coverage ClaimResponse --combine --title Claim-Coverage-ClaimResponse--FHIRAPI --url hapi.fhir.org --prefix fhir --version v1

**Comment by athiththan11** [comment]
Hi @aabuhasan 

Please find the commit for the above-reported issue [#853c1](https://github.com/athiththan11/FHIR-to-Swagger/commit/853c1ade348103a7d90ccdfabe7cc25f18f3b4e6)

Now you can use the following pattern to pass additional args

```sh
fhir-to-swagger Claim Coverage ClaimResponse --combine --title Combined--Swager --host hapi.fhir.org --base fhir --swagger-version 1.0.0
```

Hope this solves your requirement. Please test and get back.

Cheers !!!

**Comment by aabuhasan** [comment]
Hi @athiththan11 

I just tested it, thank you for your great work, but base URL "fhir" should be on all API's, so instead of "claim-api" use "fhir" on all of them.



![image](https://user-images.githubusercontent.com/13761652/70128156-152db580-1685-11ea-9222-dc8fe3d3f1e5.png)
 

![image](https://user-images.githubusercontent.com/13761652/70128410-7190d500-1685-11ea-980b-d0c0d30e09c3.png)

**Comment by athiththan11** [comment]
Hi @aabuhasan 

Please find the commit for the above-requirement [#482945c](https://github.com/athiththan11/FHIR-to-Swagger/commit/482945caf39b6634b7e986f4119aabea3db7f182)
Hope this solves.

Cheers !!!

**Comment by aabuhasan** [comment]
Hi @athiththan11 

Please remove duplicate "fhir" from request url.
![image](https://user-images.githubusercontent.com/13761652/70386720-c34a9f80-19a4-11ea-8185-e758d099d7a9.png)

**Comment by athiththan11** [comment]
Hi @aabuhasan 

Please find the enhanced fix [#9f81006](https://github.com/athiththan11/FHIR-to-Swagger/commit/9f81006ef3b346ce56d42945c7852b9ba2ce4e56)

I have introduced three new arguments for the `basePath` configurations

* `--base`: The `basePath` of the generated Swagger definition for a single resource
* `--default-base`: A default `basePath` will be applied to the generated Swagger definition for a single resource (example: `/coverage-api`)
* `--combined-base`: The `basePath` of the generated combined Swagger definition

For example: 

```
fhir-to-swagger Claim Coverage ClaimResponse --combine --title Combined--Swager --host hapi.fhir.org --combined-base fhir --default-base --swagger-version 1.0.0
```
The above command will generate a combined Swagger definition with the `basePath` `/fhir` and the resources will be starting with the following pattern as `/coverage-api/Coverage`

Hope this meets and solves your problem.
Cheers !!!
